lib/pushmi_pullyu/fedora_object_fetcher.rb

```diff
@@ -0,0 +1,59 @@
+require 'pushmi_pullyu'
+require 'net/http'
+require 'fileutils'
+
+class PushmiPullyu::FedoraObjectFetcher
```

Comment:
Yeah good suggestion. I think a namespace for PushmiPullyu::AIP:: would be a good addition for this. This helps organise all this code into a AIP folder (and you can remove all the `aip_` names too)